src/js/constraint.js

```diff
@@ -71,10 +76,12 @@ function _addOutOfBoundsClass(oob, addClasses, classes, classPrefix, outOfBounds
       oobClass = getClass('out-of-bounds', classes, classPrefix);
     }
 
-    addClasses.push(oobClass);
-    oob.forEach((side) => {
-      addClasses.push(`${oobClass}-${side}`);
-    });
+    if (oobClass) {
```

Comment:
`oobClass` should never not be defined right?

Reply:
was not clear to me, since I was able to make it be falsey in tests. Seems like a possible fail point. But if you know from what you've learned it'll never be null, I'm good to remove.

Reply:
It is set in an `if else`, so it's always either set to one value or the other. It may just be some of the required params were not passed in the test.

Reply:
doesn't totally fail I guess, but if you pass in empty arrays you get a return value of `["", "-top"]`, which seems wrong.